ckanext/datastore/backend/postgres.py

```diff
@@ -6,6 +6,10 @@
 # TODO: upstream contrib!!
 import re
 import string
+# (canada fork only): psql dump format
+# TODO: upstream contrib??
+import subprocess
+from io import DEFAULT_BUFFER_SIZE
```

Comment:
8k is small for looping over data in python code, but try it and see

Reply:
Seemed to work fine for 2GB+ files